man/systemd.unit.xml

```diff
@@ -2188,6 +2190,62 @@ PrivateTmp=yes</programlisting>
       to override the entire unit.</para>
 
     </example>
+
+    <example>
+      <title>Top level drop-ins with template units</title>
+
+      <para>Top level drop-ins can be used to change some aspect of all units
+      of a particular type. For example by creating the
+      <filename index='false'>/etc/systemd/system/service.d</filename>
```

Comment:
please suffix dirs with `/` in docs and log message, see coding style

Reply:
ack. I'll send a follow up PR to address your comments since it looks like this one got merged already